speculos/main.py

```diff
@@ -521,7 +547,7 @@ def main(prog=None) -> int:
     display_args = DisplayArgs(args.color, args.model, args.ontop, rendering,
                                args.keymap, zoom, x, y)
     server_args = ServerArgs(apdu, apirun, button, finger, seph, vnc)
-    screen_notifier = ScreenNotifier(display_args, server_args)
+    screen_notifier = ScreenNotifier(display_args, server_args, use_bagl)
```

Comment:
Instead of propagating this argument in every `DisplayNotifier` and `Display` implementations, it could make sense to add it into the `display_args: DisplayArgs` argument and accessing it via a `@decorator` in the abstract classes.